pokemongo_bot/event_handlers/telegram_handler.py

```diff
@@ -124,34 +164,57 @@ class TelegramHandler(EventHandler):
     def __init__(self, bot, config):
         self.bot = bot
         self.tbot = None
-        self.master = config.get('master', None)
+        master = config.get('master', None)
         self.pokemons = config.get('alert_catch', {})
         self.whoami = "TelegramHandler"
         self.config = config
+        if master == None:
+            self.master = None
+            return
+
+        with self.bot.database as conn:
+            # if master is not numeric, try to fetch it from the database
+            if not re.match(r'^[0-9]+$', str(master)): # master is numeric
+                self.bot.logger.info("Telegram master is not numeric: {}".format(master))
+                c = conn.cursor()
+                # do we already have a user id?
+                srchmaster = re.sub(r'^@', '', master)
+                c.execute("SELECT uid from telegram_uids where username in ('{}', '@{}')".format(srchmaster, srchmaster))
+                results = c.fetchall()
+                if len(results) > 0: # woohoo, we already saw a message from this master and therefore have a uid
+                    self.bot.logger.info("Telegram master UID from datastore: {}".format(results[0][0]))
+                    self.master = results[0][0]
+                else: # uid not known yet
+                    self.bot.logger.info("Telegram master UID not in datastore yet")
```

Comment:
Do we really need this logs on info level? Maybe its better to move it to debug?